web/src/js/components/ContentView/ContentViews.jsx

```diff
@@ -57,4 +57,14 @@ function ViewAuto({ message, flow, readonly, onChange }) {
     return <View message={message} flow={flow} readonly={readonly} onChange={onChange}/>
 }
 
-export { ViewImage, ViewRaw, ViewAuto, ViewJSON }
+function ViewServer({contentView, content}){
```

Comment:
I think we have a possible misunderstanding here. We want one button per server-side contentview - for the user, there should not be a difference between client-side and server-side views.